Cargo.toml

```diff
@@ -84,7 +84,14 @@ paste = "1.0"
 prost = "0.10.3"
 regex = { version = "1.5", optional = true }
 rustc-hex = "2.1.0"
-serde_crate = { package = "serde", version = "1", features = ["derive"], optional = true }
+# we rename the crate below because there is already a feature called `serde`,
+# so it would conflict with the implicity feature that would be added by adding
```

Comment:
S/implicity/implicit/?

Reply:
yep, don't know where the `y` came from lol will update it

Reply:
done